### PR TITLE
Implement passing arguments to @content

### LINKED ISSUE #2723: Support passing arguments to @content
See sass/sass#871

### PR DESCRIPTION
**Pull request by xzyfer**
See sass/sass#871
Spec https://github.com/sass/sass-spec/pull/1313
Fixes #2723

### CODE REVIEW

File: src/debugger.hpp [hunk 1]
```diff
@@ -605,12 +606,6 @@ inline void debug_ast(AST_Node_Ptr node, std::string ind, Env* env)
       " [hash: " << expression->hash() << "] " <<
       std::endl;
     for(const auto& i : expression->elements()) { debug_ast(i, ind + " ", env); }
-  } else if (Cast<Content>(node)) {
```
Comment (xzyfer) [inline src/debugger.hpp]: This block was a duplicate.

File: src/prelexer.cpp [hunk 2]
```diff
@@ -1260,6 +1260,9 @@ namespace Sass {
     const char* kwd_lte(const char* src) {
       return exactly<lte>(src);
     }
+    const char* kwd_using(const char* src) {
+      return keyword < using_kwd >(src);
```
Comment (glebm) [inline src/prelexer.cpp]: Can we not put spaces around `<template>`? This isn't ever necessary in C++11.

**Comment by xzyfer** [comment]
I agree it's not necessary but template space (at least between < >) is
consistent throughout much of the codebase. Can't speak for anyone else
but I've personallynd it makes the code more easily scannable.

Curious to know about your objection to the style?

File: src/prelexer.cpp (hunk 2 above)
Comment (xzyfer) [reply src/prelexer.cpp]: I agree it's not necessary but template space (at least between < >) is
consistent throughout much of the codebase. Can't speak for anyone else
but I've personally found it makes the code more easily scannable.

Curious to know about your objection to the style?

Comment (xzyfer) [reply src/prelexer.cpp]: Updated to match the adjacent style.

**Pull request merged**